### DISCUSSION TITLE
Welcome to video-subtitle-remover Discussions!

### DISCUSSION THREAD

**Discussion by YaoFANGUK** [YaoFANGUK/video-subtitle-remover#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Huyvux12** [comment]
bro can you import into google colab, my pc very poor. thanks

**Comment by nowkin** [comment]
possible to release a new version with the propainter?

**Comment by yueshenzhitan** [comment]
处理过程中发生错误: (NotFound) Cannot open file D:\vsr\视频字幕去除器\_internal\backend\models\V5\ch_det\inference.json, please confirm whether the file is normal. [Hint: Expected paddle::inference::IsFileExists(prog_file_) == true, but received paddle::inference::IsFileExists(prog_file_):0 != true:1.] (at ..\paddle\fluid\inference\api\analysis_config.cc:117)  处理不了图片
